src/instructlab/configuration.py

```diff
@@ -249,6 +252,8 @@ def get_default_config():
         ),
         serve=_serve(
             model_path=DEFAULT_MODEL_PATH,
+            host_port="127.0.0.1:8000",
+            backend=LLAMA_CPP,
```

Comment:
Why set a default since it's auto-detected? Wouldn't eval go through the same auto-detection path?